utils/import_current_vote.py

```diff
@@ -0,0 +1,40 @@
+from typing import Optional, Callable, List
+import os
+import glob
+import sys
+
+from brownie import accounts, chain
+
+from utils.config import (ldo_holder_address_for_tests,)
+
+def get_vote_script_files() -> List[str]:
+    """Return List of abs paths to vote scripts"""
+    dir_path = os.path.dirname(os.path.realpath(__file__))
+    dir_path = os.path.join(os.path.split(dir_path)[0], 'scripts')
+    vote_files = glob.glob(os.path.join(dir_path, 'vote_*.py'))
+    return vote_files
+
+
+def get_start_and_execute_votes_func() -> Optional[Callable]:
+    vote_files = get_vote_script_files()
```

Comment:
Sorry, I didn't get, why we need a closure here? Can we make it two plain stupid functions instead?

Reply:
Refactored, do you like it?